geoprocessing/src/migrations/geoprocessing/1618853324000-ExtractDistictPropsFromFeatures.ts

```diff
@@ -0,0 +1,50 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class ExtractDistictPropsFromFeatures1618853324000
+  implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    queryRunner.query(`
+ALTER TABLE features_data
+  RENAME COLUMN features_id TO feature_id;
+
+CREATE OR REPLACE FUNCTION properties_for_feature(input uuid)
+  RETURNS jsonb
+  LANGUAGE plpgsql AS
+$f$
+  DECLARE
+    _key   text;
+    _value text;
+    all_properties_across_feature_rows jsonb;
+    result jsonb;
+  BEGIN
+    result := '{}'::jsonb;
+    all_properties_across_feature_rows := (select properties from features_data where feature_id = input limit 1);
+    FOR _key, _value IN
+      SELECT * FROM jsonb_each_text(all_properties_across_feature_rows)
+    LOOP
+      result := result || jsonb_build_object(_key, array(select distinct properties -> _key from features_data where feature_id = input));
+    END LOOP;
+
+    RETURN result;
+  END
+$f$;
```

Comment:
@aagm as much as I love pg functions, I also tend to agree with folks who warn about their source control to be a bit of a parallel world to the git context of the other components, because of the indirection layer of migrations. For a few, rarely changing (I guess... 😅) functions this all looks fine to me, in any case.
However, would you have suggestions about adding some brief documentation to these pg functions? Should we add anything to the TS source in the migration? or as SQL comments within the SQL string that we pass to `queryRunner.query()`?
Again, not really a big deal here as the function is very small and rather plain and hopefully understandable at a glance, but I am open to suggestions.